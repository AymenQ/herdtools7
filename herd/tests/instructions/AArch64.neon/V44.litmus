AArch64 V44
(* Tests STP - store pair without offset *)
{ 
0:X0=x; 0:X1=y;
}

 P0                      ;
 MOVI D0, #255           ;
 MOVI D1, #255           ;
 STP D0, D1, [X0]        ;
 LDR D2, [X0, #8]        ;
 STR D2, [X1]            ;
 exists (x=255 /\ y=255)
