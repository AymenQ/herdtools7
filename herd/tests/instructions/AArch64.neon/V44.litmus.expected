Test V44 Allowed
States 1
x=255; y=255;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x=255 /\ y=255)
Observation V44 Always 1 0
Hash=db4fce50ab65a58badf62dc7e5516861

